loopchain/channel/channel_service.py

```diff
@@ -745,21 +664,41 @@ def score_invoke(self, _block: Block) -> dict or None:
             }
             transactions.append(transaction)
 
-        request = {
+        request_origin = {
             'block': {
                 'blockHeight': _block.header.height,
                 'blockHash': _block.header.hash.hex(),
                 'prevBlockHash': _block.header.prev_hash.hex() if _block.header.prev_hash else '',
                 'timestamp': _block.header.timestamp
             },
-            'transactions': transactions
+            'transactions': transactions,
+            'prevBlockGenerator': prev_block.header.peer_id.hex_hx() if prev_block.header.peer_id else '',
+            'prevBlockValidators': [rep['id'] for rep in self.__peer_manager.get_reps()]
         }
-        request = convert_params(request, ParamType.invoke)
+
+        if conf.ENABLE_IISS:
+            request_origin['isBlockEditable'] = hex(conf.ENABLE_IISS)
+
+        request = convert_params(request_origin, ParamType.invoke)
         stub = StubCollection().icon_score_stubs[ChannelProperty().name]
-        response = stub.sync_task().invoke(request)
+        response: dict = cast(dict, stub.sync_task().invoke(request))
         response_to_json_query(response)
 
-        tx_receipts = response["txResults"]
+        if request_origin['transactions']:
+            utils.logger.notice(f"in score invoke \nrequest({request_origin}) \nresponse({response})")
```

Comment:
if this line of log is necessary, how about re-locate after below line?

https://github.com/icon-project/loopchain/blob/819e8246d3883533190b96a5e6ac4482f4e060fd/loopchain/channel/channel_service.py#L657-L665